index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css" integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Front End Development</title>
+</head>
+<body>
+    <section class="container-1">
+        <header class="header">
+            <h3>Tapish Khari</h3>
+            <nav><i class="fa-solid fa-bars"></i></nav>
+        </header>
+        <div class="section">
+            <h1>Hey There.
+                I’m Tapish</h1>
+            <h2>I am a Software Developer</h2>
+            <p>
+        I can help you build a product, feature
+        or website look through some of my work and experience!
+        if you like what you see and have a project you need coded.
+        don't hesitate to contact me.
+            </p>
+            <ul>
+                <li><a href="#"><img src="images/Github.png" alt="github-icon"></a> </li>
+                <li><a href="#"><img src="images/linkedin.png" alt="linked-icon"></a> </li>
+                <li><a href="#"><img src="images/chill.png" alt="chill-icon"></a> </li>
+                <li><a href="#"><img src="images/twitter.png" alt="twitter-icon"></a> </li>
+                <li><a href="#"><img src="images/Medium.png" alt="Medium-icon"></a> </li>
+            </ul>
+        </div>
+    </section>
+</body>
+</html>
```

Comment:
- [ ] Good job so far you did a nice job :+1:  but your website text looks different from the Figma design :cry: Please use your headline section with the same font and color properties as defined for the template to make it pretty and it will be a wonderful website. :smile: 
Check the screenshots below to see the differences :arrow_down: 

#### Your design
![Screenshot from 2022-08-25 10-25-52](https://user-images.githubusercontent.com/6615394/186643003-fc8b0d85-169a-4b82-8d8f-11ffb182446e.png)

#### Figma design
![Screenshot from 2022-08-25 10-24-15](https://user-images.githubusercontent.com/6615394/186643011-7cf9151f-7ef7-4d50-ba14-a67ea16928b1.png)

Reply:
@codecaiine I have made the changes as per the requirement.